File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/objects/event/EventTypeTest.java
package com.github.dedis.popstellar.model.objects.event;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.MatcherAssert.assertThat;

public class EventTypeTest {

  @Test
  public void getSuffixTest() {
    assertThat(EventType.MEETING.suffix, is("M"));
    assertThat(EventType.ROLL_CALL.suffix, is("R"));
    assertThat(EventType.POLL.suffix, is("P"));
    assertThat(EventType.DISCUSSION.suffix, is("D"));
  }
}

File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/objects/event/EventTypeTest.kt
package com.github.dedis.popstellar.model.objects.event

import org.hamcrest.CoreMatchers
import org.hamcrest.MatcherAssert
import org.junit.Test

class EventTypeTest {
  @Test
  fun suffixTest() {
    MatcherAssert.assertThat(EventType.MEETING.suffix, CoreMatchers.`is`("M"))
    MatcherAssert.assertThat(EventType.ROLL_CALL.suffix, CoreMatchers.`is`("R"))
    MatcherAssert.assertThat(EventType.POLL.suffix, CoreMatchers.`is`("P"))
    MatcherAssert.assertThat(EventType.DISCUSSION.suffix, CoreMatchers.`is`("D"))
  }
}
